src/features/router/Router.tsx

```diff
@@ -11,21 +11,31 @@ import Wallet from "features/wallet";
 import Footer from "common/components/footer";
 import { SigningKeys } from "App";
 
+import useVoteData, { PriceRequestRound } from "common/hooks/useVoteData";
+
 interface Props {
   signingKeys: SigningKeys;
 }
 
 const Router: FC<Props> = ({ signingKeys }) => {
+  const {
+    data: voteSummaryData = [] as PriceRequestRound[],
+    refetch: refetchVoteSummaryData,
+  } = useVoteData();
+
   return (
     <BrowserRouter>
       <div>
         <Navbar />
         {/* A <Switch> looks through its children <Route>s and
             renders the first one that matches the current URL. */}
-        <Wallet signingKeys={signingKeys} />
+        <Wallet
+          signingKeys={signingKeys}
+          refetchVoteSummaryData={refetchVoteSummaryData}
+        />
         <Switch>
           <Route path="/">
-            <Vote signingKeys={signingKeys} />
+            <Vote signingKeys={signingKeys} voteSummaryData={voteSummaryData} />
```

Comment:
just curious why this was moved to a prop

Reply:
Didn't want to recreate the hook in two locations, when I was just passing down a refetch.